daq-2.0.6/Makefile

```diff
@@ -195,24 +206,24 @@ distuninstallcheck_listfiles = find . -type f -print
 am__distuninstallcheck_listfiles = $(distuninstallcheck_listfiles) \
   | sed 's|^\./|$(prefix)/|' | grep -v '$(infodir)/dir$$'
 distcleancheck_listfiles = find . -type f -print
-ACLOCAL = ${SHELL} /users/graceliu/git/onvm-snort/daq-2.0.6/missing aclocal-1.14
+ACLOCAL = ${SHELL} /home/student/sources/onvm-snort/daq-2.0.6/missing aclocal-1.15
```

Comment:
Should `/home/student/sources` be set to a shell variable so that it is correct for any given user?

Reply:
I believe the Makefile is auto-generated using the configure script which contains the shell variable so that it is correct for any given user